### PR TITLE
Add list of jobs to scheduler UI

### PR DESCRIPTION
**Pull request by andygrove**
# Which issue does this PR close?

N/A

 # Rationale for this change

It's nice to be able to see jobs running!

progress2

# What changes are included in this PR?

- REST endpoint `/api/jobs`
- REST endpoint `/api/job/<id>/summary`
- List of jobs in UI

# Are there any user-facing changes?

### CODE REVIEW

**Comment by andygrove** [comment]
cc @msathis - the UI looks great .. thanks for your work to get this started!

File: ballista/ui/scheduler/src/App.tsx [hunk 1]
```diff
@@ -24,46 +24,9 @@ import { Footer } from "./components/Footer";
 
 import "./App.css";
 
-function uuidv4() {
```
Comment (avantgardnerio) [inline ballista/ui/scheduler/src/App.tsx:27]: wat?? Glad to see this function go.

**Review by avantgardnerio — APPROVED** [review]
Love it!

**Comment by andygrove** [comment]
Thanks for the review @avantgardnerio!

**Pull request merged**